peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -164,9 +164,15 @@ <h1>{{ document.title }}</h1>
                       {% block document-metadata-content-alternative-names %}
                         {% with document.alternative_names.all as alternative_names %}
                           {% if alternative_names %}
-                            <dt>
-                              {% trans 'Alternative names' %}
-                            </dt>
+                            {% if document.doc_type == "judgment" %}
+                              <dt>
+                                {% trans 'Law Report Citations' %}
```

Comment:
```suggestion
                                {% trans 'Law report citations' %}
```

We use "Sentence case" rather than Title Case.